packages/palette/src/elements/Grid/Grid.story.tsx

```diff
@@ -0,0 +1,29 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { BorderBox } from "../BorderBox"
+import { Col, Grid, Row } from "./Grid"
+
+storiesOf("Components/Grid", module).add("Basic", () => {
+  return (
+    <Grid
+      border={{
+        default: "4px solid pink",
```

Comment:
So `default` can actually be anything and the behavior is a bit unintuitive. Basically if it doesn't find a key that's a valid alias it'll use that value as the default. And if you have multiple missing keys, whichever is last in the object wins.